public/index.html

```diff
@@ -37,24 +21,16 @@
         firebase.auth.FacebookAuthProvider.PROVIDER_ID,
         firebase.auth.TwitterAuthProvider.PROVIDER_ID,
         firebase.auth.GithubAuthProvider.PROVIDER_ID,
-        firebase.auth.EmailAuthProvider.PROVIDER_ID,
-        //firebase.auth.PhoneAuthProvider.PROVIDER_ID,
-        //firebaseui.auth.AnonymousAuthProvider.PROVIDER_ID
```

Comment:
Should we remove commented out code? It's better isn't it?

Reply:
You're comment is on the wrong side, the other side with the green lines are the new changes. You'd be pleased to see I removed the comments. I am not against leaving commented out code, up to point ...